code/modules/games/cas.dm

```diff
@@ -0,0 +1,120 @@
+// CARDS AGAINST SPESS
+// This is a parody of Cards Against Humanity (https://en.wikipedia.org/wiki/Cards_Against_Humanity)
+// which is licensed under CC BY-NC-SA 2.0, the full text of which can be found at the following URL:
+// https://creativecommons.org/licenses/by-nc-sa/2.0/legalcode
+// Original code by Zuhayr, Polaris Station, ported with modifications
+
+/obj/item/toy/cards/deck/cas
+	name = "\improper CAS deck (white)"
+	desc = "A deck for the game Cards Against Spess, still popular after all these centuries. Warning: may include traces of broken fourth wall.\nThis is the white deck."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "deck_caswhite_full"
+	deckstyle = "caswhite"
+	var/blanks = 0
+	var/decksize = 100
```

Comment:
decks have unlimited size, this is just for generating cards it contains initially